libs/core/src/lib/dialog/dialog-default/dialog-default.component.html

```diff
@@ -18,18 +19,22 @@ <h1 fd-title>{{ _defaultDialogContent?.title }}</h1>
 
     <fd-dialog-footer *ngIf="_showFooter()">
         <fd-button-bar
-                *ngIf="_defaultDialogContent?.approveButton"
-                fdType="emphasized"
-                fd-initial-focus
-                [label]="_defaultDialogContent?.approveButton"
-                [compact]="!_defaultDialogConfiguration.mobile"
-                (click)="_approveButtonClicked()">
+            *ngIf="_defaultDialogContent?.approveButton"
+            fdType="emphasized"
+            fd-initial-focus
+            [label]="_defaultDialogContent?.approveButton"
+            [compact]="!_defaultDialogConfiguration.mobile"
+            (click)="_approveButtonClicked()"
+            [ariaLabel]="_defaultDialogContent?.approveButton + ' emphasized button'"
```

Comment:
i18n support? it would be better if we could find a way not to hardcode this at all. `button` will get read by JAWS so it may not even be needed here.

Reply:
fixed